guessthenumber/GuessGame.java

```diff
@@ -0,0 +1,39 @@
+package guessthenumber;
+
+import java.util.Random;
+
+public class GuessGame {
+
+    public void startGame() {
+        Random rand = new Random();
+        int computerGeneratedNumber = rand.nextInt(1, 3);
+        System.out.println("Computer generated number: " + computerGeneratedNumber);
+
+        Player player1 = new Player();
+        Player player2 = new Player();
+        Player player3 = new Player();
+
+        player1.setName("Tilica");
+        player2.setName("Bambilica");
+        player3.setName("Marcela");
+
+        int player1Guess = player1.getPlayerGuessNumber();
+        int player2Guess = player2.getPlayerGuessNumber();
+        int player3Guess = player3.getPlayerGuessNumber();
+
+        boolean isGameOn = true;
+
+        while (isGameOn) {
+            if (player1Guess == computerGeneratedNumber) {
```

Comment:
With this logic, we will always print just one player, but if 2 or 3 players guess `computerGeneratedNumber`, we should find another construction and print them as well.
We can have all the checks inside of a single if, like: 
`if(p1 == compGen || p2 == compGen || p3 == compGen) {
   //code goes here
   //print all the players who guessed computerGeneratedNumber
}`

When we will learn about Map collection, we will discover that we can use that to have a `key - value` pair, in which we can store the player with the guessed number, in situations like this.